pyod/models/auto_encoder_torch.py

```diff
@@ -44,52 +44,74 @@ def __getitem__(self, idx):
         return torch.from_numpy(sample), idx
 
 
-class inner_autoencoder(nn.Module):
+class InnerAutoencoder(nn.Module):
     def __init__(self,
                  n_features,
-                 hidden_neurons=[128, 64],
+                 hidden_neurons=(128, 64),
```

Comment:
Python functions should never have mutable default arguments (as they live in global scope and are only initialized once).